docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`7052` Add support for Monerium EUR emoney (EURe) for Ethereum, Polygon, and Gnosis chains.
```

Comment:
remove the changelog entry and the "Closes XXX" comment. This is just a part of the issue. The other part is using the monerium API.